src/legacy/legacy-grpc-network-client.web.ts

```diff
@@ -119,39 +126,45 @@ class LegacyGRPCNetworkClient implements LegacyNetworkClient {
     })
   }
 
-  /* eslint-disable class-methods-use-this */
-  public XRPAmount(): XRPAmount {
-    return new XRPAmount()
-  }
-
-  public Payment(): Payment {
-    return new Payment()
-  }
-
-  public Transaction(): Transaction {
-    return new Transaction()
-  }
-
-  public SubmitSignedTransactionRequest(): SubmitSignedTransactionRequest {
-    return new SubmitSignedTransactionRequest()
-  }
-
-  public GetLatestValidatedLedgerSequenceRequest(): GetLatestValidatedLedgerSequenceRequest {
-    return new GetLatestValidatedLedgerSequenceRequest()
-  }
-
-  public GetTransactionStatusRequest(): GetTransactionStatusRequest {
-    return new GetTransactionStatusRequest()
-  }
-
-  public GetAccountInfoRequest(): GetAccountInfoRequest {
-    return new GetAccountInfoRequest()
-  }
-
-  public GetFeeRequest(): GetFeeRequest {
-    return new GetFeeRequest()
+  public async createSignedTransaction(
```

Comment:
Also: Can we dedupe this with the other class?

Reply:
I'll look into it, this would be a great improvement. The code here looks highly duplicative but the problem is that all the types here are technically different.